lms/envs/common.py

```diff
@@ -434,6 +434,16 @@
     'ENABLE_FOOTER_MOBILE_APP_LINKS': False,
 
     # Let students save and manage their annotations
+    # .. toggle_name: FEATURES['ENABLE_EDXNOTES']
+    # .. toggle_implementation: SettingToggle
+    # .. toggle_default: False
+    # .. toggle_description: The edxnotes app is responsible for displaying parts of the Notes UI to students in different parts
+    #   of the LMS, as well as figuring out whether Notes is enabled for a particular situation. The bulk of the actual work in
+    #   storing the notes is done by a separate service (see the edx-notes-api repo).
+    # .. toggle_warnings: Requires the edx-notes-api service properly running and to have configured the django settings as well
```

Comment:
Could you specify which are those settings? So far, I think that they are EDXNOTES_INTERNAL_API and EDXNOTES_INTERNAL_API

Reply:
Thank you @morenol!!. Great I think EDXNOTES_INTERNAL_API and EDXNOTES_PUBLIC_API am I right?!